src/components/ReviewFormContainer/index.tsx

```diff
@@ -121,49 +121,57 @@ function ReviewFormContainer({ csrfToken }) {
    * toggling instead of an anchor element. This is for the Personal and the
    * Account sections.
    */
-  const editSectionButton = (editSectionFlag, sectionName, page) =>
-    clientSide ? (
+  const editSectionButton = (editSectionFlag, sectionName, page) => {
+    const label = `${t("button.edit")} ${page === "personal" ? t(`review.section.personal`) : t("review.createAccount")}`;
+    return clientSide ? (
       <Button
         variant="primary"
         id={`editSectionButton-${sectionName}`}
```

Comment:
This is causing invalid id values for some of the buttons because some of the section names contain spaces.

Reply:
Was an update made to resolve this?